README.md

```diff
@@ -139,6 +138,19 @@ telegraf --config telegraf.conf
 telegraf --config telegraf.conf --input-filter cpu:mem --output-filter influxdb
 ```
 
+## Contributing
+
+There are multiple great ways to get involved with the Telegraf project, from contributing code to sharing your story on how you use it.
+
+- Fix and [report bugs](https://github.com/influxdata/telegraf/issues/new)
+- [Improve documentation](https://github.com/influxdata/telegraf/issues?q=is%3Aopen+label%3Adocumentation)
```

Comment:
Good catch! Maybe instead of this list, we could have this section be similar to what influxdb's project has something like this: https://github.com/influxdata/telegraf/blob/e99e3eb75661ec19684c45248fda87582113709b/README.md#contribute-to-the-project

Reply:
Great, its been updated to match.